src/opm/input/eclipse/EclipseState/Grid/NNC.cpp

```diff
@@ -171,14 +177,81 @@ bool is_neighbor(const EclipseGrid& grid, std::size_t g1, std::size_t g2) {
         }
     }
 
+    void NNC::load_editr(const EclipseGrid& grid, const Deck& deck) {
+        // Will contain EDITNNCR data in reverse order to be able
+        // use unique to only keep the last one from the data file.
+        std::deque<NNCdata> nnc_editr;
+
+        const auto& keyword_list = deck.getKeywordList<ParserKeywords::EDITNNCR>();
+
+        if (keyword_list.empty()) {
+            return;
+        }
+
+        for (const auto& keyword_ptr : deck.getKeywordList<ParserKeywords::EDITNNCR>()) {
+            const auto& records = *keyword_ptr;
+            if (records.empty()) {
+                continue;
+            }
+            for (const auto& record : records) {
+                auto index_pair = make_index_pair(grid, record);
+                if (!index_pair)
+                    continue;
+
+                auto [g1, g2] = index_pair.value();
+                if (is_neighbor(grid, g1, g2))
+                    continue;
+
+                double trans = record.getItem(6).getSIDouble(0);
+                nnc_editr.emplace_front(g1, g2, trans);
+            }
+
+            if (!this->m_editr_location)
+                this->m_editr_location = keyword_ptr->location();
```

Comment:
With multiple EDITNNCR keywords, this will only point to the last. This is the same as the existing behaviour for NNC and EDITNNC, so I guess I will not require a refactoring of this now...

Reply:
Yes that is a common problem. It will be the first occurrence, though.